ui/src/pages/frontsite/Documents/DocumentsDetails/DocumentCirculation/index.js

```diff
@@ -5,6 +5,7 @@ import { showTab } from '../state/actions';
 const mapStateToProps = state => ({
   documentDetails: state.documentDetailsFront.data,
```

Comment:
I would pass the data documentDetails from the parent component instead of redux, and here just pass (rename) `loanRequestIsLoading`

Reply:
@ntarocco could you explain why you prefer the props passing over redux in this case?